web/src/main/kotlin/no/nav/su/se/bakover/web/routes/Feilresponser.kt

```diff
@@ -61,4 +63,26 @@ internal object Feilresponser {
         "Ugyldig input",
         "ugyldig_input"
     )
+
+    internal fun SimuleringFeilet.tilResultat(): Resultat {
```

Comment:
Nevnte du ikke en tilsvarende funksjon for UtbetalingFeilet?

Reply:
Nei, var SimuleringFeilet jeg nevnte (ihvertfall den jeg tenkte på), men kan sjekke om vi burde gjøre det samme for UtbetalingFeilet. Tror kanskje den ikke brukes like ofte som simuleringsfeil.